root/usr/sbin/hotsync

```diff
@@ -126,9 +127,22 @@ if [ "${1:-unset}" == '--dry-run' ]; then
     XTRA_OPTS="--dry-run -v"
 fi
 
-#launch backup before rsync
+# launch backup before rsync
 /sbin/e-smith/backup-config
 
+# keep only existing, readable files and directories in include list
+TMPFILE=$(mktemp)
```

Comment:
is it possible to avoid tmp file using a variable ?

Reply:
done